packages/neuron-ui/src/containers/Navbar/index.tsx

```diff
@@ -147,6 +155,46 @@ const Navbar = () => {
     bestBlockNumber >= 0 ? localNumberFormatter(bestBlockNumber) : '-'
   }`
 
+  useEffect(() => {
+    intervalHandle.current = setTimeout(() => {
```

Comment:
I'll optimize the logic to clear the Timeout before setting a new one, to avoid memory leaks.  
Using `setTimeout` may be better at this moment, when using setInterval, the execution time of the timer is fixed, and the next callback function will be called after the set time interval, regardless of whether the previous callback function has been completed. This may cause the execution time of the timer to be inaccurate due to competition with other codes.